tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/loc/LineOfCodeTest.java

```diff
@@ -0,0 +1,31 @@
+package org.ballerinalang.test.loc;
+
+import org.ballerinalang.launcher.util.BCompileUtil;
+import org.ballerinalang.launcher.util.BRunUtil;
+import org.ballerinalang.launcher.util.CompileResult;
+import org.ballerinalang.model.values.BInteger;
+import org.ballerinalang.model.values.BValue;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
```

Comment:
Test class needs test description and class comments.

Reply:
Added with https://github.com/ballerina-platform/ballerina-lang/pull/9987/commits/e83a124f8e8b1974e2b3e9c10e798c4bc4393f2b